app/models/persistent_attachments/va_form.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class PersistentAttachments::VAForm < PersistentAttachment
+  attr_accessor :form_id
+
+  include ::FormUpload::Uploader::Attachment.new(:file)
+
+  before_destroy(:delete_file)
+
+  def max_pages
+    if form_id == '21-0779'
+      4
+    else
+      10
+    end
+  end
+
+  def min_pages
+    if form_id == '21-0779'
+      2
+    else
+      1
+    end
+  end
+
```

Comment:
This would be better as a collection of defaults, like so:

```ruby
presents = [{
  '21-0779': {
    max_pages: 4,
    min_pages: 2
  }
}]
```